docker/dockerfiles/toolchains/toolchains-centos7.Dockerfile

```diff
@@ -70,7 +70,7 @@ LABEL org.opencontainers.image.source="https://github.com/starrocks/starrocks"
 
 RUN yum-config-manager --add-repo https://cli.github.com/packages/rpm/gh-cli.repo && yum install -y gh && \
         yum install -y ${EPEL_RPM_URL} && yum install -y wget unzip bzip2 patch bison byacc flex autoconf automake make \
-        libtool which git ccache binutils-devel python3 file java-11-openjdk java-11-openjdk-devel java-11-openjdk-jmods less psmisc && \
+        libtool which git ccache binutils-devel python3 file java-17-openjdk java-17-openjdk-devel java-17-openjdk-jmods less psmisc && \
```

Comment:
jdk17 is not there in centos7 yum repository. It is not good enough just bumping java-11 to java-17

Reply:
So what's your advice? Download from source?

Reply:
if can't find any yum repository to download the jdk17, it makes no option but downloads from official site.